errorreporting/errorreporting_quickstart/main.go

```diff
@@ -32,30 +33,34 @@ var errorClient *errorreporting.Client
 func main() {
 	ctx := context.Background()
 
-	// Sets your Google Cloud Platform project ID.
-	projectID := "YOUR_PROJECT_ID"
+	// TODO: Sets your Google Cloud Platform project ID via environment or explicitly
+	projectID := os.Getenv("GOOGLE_PROJECT_ID")
+	if projectID == "" {
+		projectID = "YOUR_PROJECT_ID"
+	}
 
 	var err error
 	errorClient, err = errorreporting.NewClient(ctx, projectID, errorreporting.Config{
-		ServiceName: "myservice",
+		ServiceName:    "errorreporting_quickstart",
+		ServiceVersion: "0.0.0",
 		OnError: func(err error) {
-			log.Printf("Could not log error: %v", err)
+			log.Printf("Could not repport the error: %v", err)
 		},
 	})
 	if err != nil {
 		log.Fatal(err)
 	}
 	defer errorClient.Close()
 
-	resp, err := http.Get("not-a-valid-url")
+	err = errors.New("Something went wrong")
 	if err != nil {
 		logAndPrintError(err)
 		return
 	}
-	log.Print(resp.Status)
 }
 
 func logAndPrintError(err error) {
+	// error context is autopopulated
```

Comment:
nit: use complete sentences for comments.

Reply:
English is not my mother tongue. To the best of my knowledge it is a complete sentence. Can you provide your version to demonstrate what you mean?

Reply:
How about 
```suggestion
	// The context of the error is auto-populated.
```

Reply:
Yes, sorry, I just meant that go style is to capitalize and use a period. See https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Thanx